packages/edit-widgets/src/store/test/actions.js

```diff
@@ -178,6 +178,24 @@ describe( 'actions', () => {
 				},
 			} );
 
+			expect(
+				saveWidgetAreasGen.next()
+			).toEqual( {
+				done: false,
+				value: {
+					type: 'DISPATCH',
+					storeKey: 'core/notices',
+					actionName: 'createSuccessNotice',
+					args: [
+						'Block areas saved succesfully.',
+						{
+							id: 'WIDGET_AREAS_SAVE_NOTICE_ID',
```

Comment:
Nit pick: Maybe the test can import WIDGET_AREAS_SAVE_NOTICE_ID as a constant from the actions file.

Reply:
It's not exported at the moment. I mean I could export it but for a test, maybe it's not that important.